### DISCUSSION TITLE
Search operators / Advanced search?

### DISCUSSION THREAD

**Discussion by rguzmanl** [netbox-community/netbox#5607] [Q&A] [answered]
Hi,

I am new working with NetBox. I've been looking around here and in the documentation to see if there is and "advanced search" feature.

A user asked for a search functionality capable of looking/filtering by: "starts with", "contains", "ends with", "is same", etc.

**Comment by ryanmerolle** [comment, marked as answer]
There is none currently, but "contains" is the default behavior for all text based search boxes.
